weave/wandb_interface/wandb_lite_run.py

```diff
@@ -228,9 +232,14 @@ def log_artifact(
                 aliases=["latest"] + additional_aliases,
                 use_after_commit=False,
             )  # type: ignore
-            # wait for the artifact to be committed before returning the result
-            if future is not None:
-                future.result()
+
+            if wandb_ver < parse_version("0.16.4"):
```

Comment:
we dont need to worry about backcompat this much, I'd say just change the code back to what it was before and pin wandb in requirements to be >=0.16.4

Reply:
done
https://github.com/wandb/weave/pull/1301/files/4bebf967a66aa5b26613b7c4adf0113d2edb02df..15efd2e01b9b05d6a0bdef3250f78523de0fbaaf